snippets/distr-tracing-tempo-tempomonolithic-custom-resource.adoc

```diff
@@ -11,46 +11,47 @@ kind: TempoMonolithic # <1>
+    tls: # <9>
       enabled: true
-      caName: <ca_certificate_configmap_name> # <9>
+      caName: <ca_certificate_configmap_name> # <10>
   jaegerui:
-    enabled: true # <10>
+    enabled: true # <11>
     route:
-      enabled: true # <11>
-  resources: # <12>
+      enabled: true # <12>
+  resources: # <13>
     total:
       limits:
         memory: <value>Gi
         cpu: <value>m
   multitenancy:
     enabled: true
     mode: openshift
-    authentication: # <13>
-      - tenantName: dev # <14>
-        tenantId: "1610b0c3-c509-4592-a256-a1871353dbfa" # <15>
+    authentication: # <14>
+      - tenantName: dev # <15>
+        tenantId: "1610b0c3-c509-4592-a256-a1871353dbfa" # <16>
       - tenantName: prod
         tenantId: "1610b0c3-c509-4592-a256-a1871353dbfb"
 ----
 <1> This CR creates a TempoMonolithic deployment with trace ingestion in the OTLP protocol.
 <2> The namespace that you have chosen for the TempoMonolithic deployment.
-<3> Specifies the storage for storing traces.
-<4> Type of storage for storing traces: in-memory storage, a persistent volume, or object storage. The value for a persistent volume is `pv`. The accepted values for object storage are `s3`, `gcs`, or `azure`, depending on the used object store type. The default value is `memory` for the `tmpfs` in-memory storage, which is only appropriate for development, testing, demonstrations, and proof-of-concept environments because the data does not persist when the pod is shut down.
-<5> Memory size: For in-memory storage, this means the size of the `tmpfs` volume, where the default is `2Gi`. For a persistent volume, this means the size of the persistent volume claim, where the default is `10Gi`. For object storage, this means the size of the persistent volume claim for the Tempo Write-Ahead Logging (WAL), where the default is `10Gi`.
-<6> Optional: For object storage, the type of object storage. The accepted values are `s3`, `gcs`, and `azure`, depending on the used object store type.
-<7> Optional: For object storage, the value of the `name` in the `metadata` of the storage secret. The storage secret must be in the same namespace as the TempoMonolithic instance and contain the fields specified in "Table 1. Required secret parameters" in the section "Object storage setup".
-<8> Optional.
-<9> Optional: Name of a `ConfigMap` object that contains a CA certificate.
-<10> Exposes the Jaeger UI, which visualizes the data, via a route.
-<11> Enables creation of a route for the Jaeger UI.
-<12> Optional.
-<13> Lists the tenants.
-<14> The tenant name from the `X-Scope-OrgId` header when ingesting the data.
-<15> The unique identifier of the tenant. Must be unique throughout the lifecycle of the TempoMonolithic deployment. This ID will be added as a prefix to the objects in the object storage. You can reuse the value of the UUID or `tempoName` field.
+<3> Red{nbsp}Hat supports only the custom resource options that are available in the {DTProductName} documentation.
```

Comment:
This line and a similar line in the other file is the only content added. Git shows the rest of the changes as new content only due to the callout renumbering.